tests/queries/0_stateless/01883_with_grouping_sets.reference

```diff
@@ -73,6 +73,19 @@
 2	10	5400
 
 0	0	49500
+SELECT
+    fact_1_id,
+    fact_3_id,
+    sum(sales_value) AS sales_value
+FROM grouping_sets
+GROUP BY
+    GROUPING SETS (
+        (fact_1_id),
+        (fact_1_id, fact_3_id))
```

Comment:
What do you think about improving the formatting with one element?

Right now (including this changes) it includes extra whitespace:

```sql
SELECT *
FROM data
GROUP BY
    GROUPING SETS ( (foo))
```

Either this:

```sql
SELECT *
FROM data
GROUP BY
    GROUPING SETS ((foo))
```

Or this:

```sql
SELECT *
FROM data
GROUP BY
    GROUPING SETS (
        (foo)
    )
```

Reply:
Thank you! I also cannot stand extra whitespace :)

Reply:
Second example is ok.